### PR TITLE
fix(quick-add): Fixed quick add with right WI type

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

**Comment by rgarg1** [comment]
All looks good @sanbornsen. A few changes that remain are:

1) 'Planner Item' WIT should be changed to 'Task'
2) 'User Story' WIT is not needed
3) Icons for Scenario, Feature, and Value Proposition do not match to the ones in production

**Comment by sanbornsen** [comment]
@rgarg1 I see planner item in the original response. What I am doing instead, is taking the original set of work item types and putting the JSON in our mock. I think, that will solve all the problems.

**Comment by rgarg1** [comment]
Pls go ahead @nimishamukherjee . If there are more changes, those could be addressed later. These changes are good.

**Comment by sanbornsen** [comment]
I see the functional tests are failing. Let me fix them first.

**Comment by sanbornsen** [comment]
[test]

**Comment by sanbornsen** [comment]
That seems to be a random failure, can't detect the reason. We are good to go for now!

File: src/app/mock/mock-data/schema-mock-generator.ts [hunk 1]
```diff
@@ -86,683 +86,7 @@ export class SchemaMockGenerator {
...
..."system.description":{"description":"A descriptive text of the work item","label":"Description","required":false,"type":{"kind":"markup"}},"system.iteration":{"description":"The iteration to which the work item belongs","label":"Iteration","required":false,"type":{"kind":"iteration"}},"system.order":{"description":"Execution Order of the workitem.","label":"Execution Order","required":false,"type":{"kind":"float"}},"system.remote_item_id":{"description":"The ID of the remote work item","label":"Remote item","required":false,"type":{"kind":"string"}},"system.state":{"description":"The state of the work item","label":"State","required":true,"type":{"baseType":"string","kind":"enum","values":["new","open","in progress","resolved","closed"]}},"system.title":{"description":"The title text of the work item","label":"Title","required":true,"type":{"kind":"string"}},"system.updated_at":{"description":"The date and time when the work item was last updated","label":"Updated at","required":false,"type":{"kind":"instant"}}},"icon":"fa fa-university","name":"Fundamental","updated-at":"2017-07-26T20:27:36.310986Z","version":0},"id":"ee7ca005-f81d-4eea-9b9b-1965df0988d0","type":"workitemtypes"}];
```
Comment (nimishamukherjee) [inline src/app/mock/mock-data/schema-mock-generator.ts]: Can this be formatted so that it is in a human readable format?

**Comment by rgarg1** [comment]
Are the tests passing now? I'd suggest you run all func tests, and not just the smoke as part of this fix.

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2091-11 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2091-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2091-12 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2091-fabric8-planner.badger.fabric8.io

**Pull request merged**